components/footer/Footer.tsx

```diff
@@ -49,26 +49,49 @@ export interface RegionOptions {
   language?: Item[];
 }
 
+export interface NewsletterBanner {
+  /** @description desktop otimized image */
+  desktop?: ImageWidget;
+  /** @description mobile otimized image */
+  mobile?: ImageWidget;
+  /** @description Image's alt text */
+  alt?: string;
+  action?: {
+    /** @description when user clicks on the image, go to this link */
+    href: string;
+    /** @description Image text title */
+    title: string;
+    /** @description Image text subtitle */
+    subTitle: string;
+    /** @description Button label */
+    label: string;
+  };
+}
+
 export interface NewsletterForm {
   placeholder?: string;
   buttonText?: string;
   /** @format html */
   helpText?: string;
+  /** @format color */
+  backgroundColor?: string;
+  /** @format color */
+  colorText?: string;
```

Comment:
Para esse tipo de atributo, é interessante deixá-lo semanticamente próximo à propriedade CSS.
```suggestion
  color?: string;
```